docs/root/configuration/upstream/cluster_manager/cluster_runtime.rst

```diff
@@ -135,8 +135,8 @@ upstream.healthy_panic_threshold
   Defaults to 50%.
 
 upstream.use_http2
-  Whether the cluster utilizes the *http2* :ref:`protocol options <envoy_v3_api_field_config.cluster.v3.Cluster.http2_protocol_options>`
-  if configured. Set to 0 to disable HTTP/2 even if the feature is configured. Defaults to enabled.
+  Whether the cluster utilizes the *http2* if configured in `HttpProtocolOptions <envoy_v3_msg_config.filters.network.http_connection_manager.v3.HttpProtocolOptions>`.
+  Set to 0 to disable HTTP/2 even if the feature is configured. Defaults to enabled. FIXME
```

Comment:
This was present from way, way back when H2 was originally implemented. IMO I would just deprecate/delete this runtime field (perhaps as a different PR).